packages/block-editor/src/components/iframe/index.js

```diff
@@ -182,8 +182,11 @@ function Iframe( { contentRef, children, head, ...props }, ref ) {
 				contentRef.current = body;
 			}
 
+			if ( ! window.__disableEventBubbling ) {
+				bubbleEvents( contentDocument );
```

Comment:
There's no other way to prevent the tracing tool from also tracing the events we bubble. Ideally we remove this soon. I have an idea for this which shouldn't be hard to do.